test/es6/rlexe.xml

```diff
@@ -95,13 +95,15 @@
       <tags>BugFix</tags>
     </default>
   </test>
+  <!-- Disabling for now because the test is flaky and fails in some situations
```

Comment:
Is there a bug filed for this? 
It seems out of scope of this PR, although I understand you wanted all tests to pass to test the build process properly.